src/Components/CreateDog/CreateDogList.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useEffect, useState } from "react";
+import { connect, useDispatch } from "react-redux";
+import "./CreateDogList.styles.scss";
+import axios from "axios";
+import { setLoading } from "../../Redux/Loading/LoadingSlice";
+import Loading from "../Loading/Loading";
+import InfiniteScroll from "react-infinite-scroll-component";
+import CardTemplate from "../DogCardTemplate/DogCardTemplate";
+import urls from "../../const/urls";
+
+const CreateDogList = ({ isLoading, setLoading }) => {
+  const [dogNames, setDogNames] = useState([]);
+  const [dogInfos, setDogInfos] = useState([]);
+  const [combinedDogData, setCombinedDogData] = useState([]);
+  const dispatch = useDispatch();
+  const setLoadingTrue = () => { dispatch(setLoading(true)); };
+  const hasMore = useState(true);
+  const [offsetAmount, setOffsetAmount] = useState(0);
+  const limitperPage = 20;
+  const dogUrl = `${urls.dogNinjaApi}+&offset=${offsetAmount}`;
```

Comment:
why `+&` ? Usually it is `?param1=param1Value&param2=param2Value`

Reply:
In the endpoint URL file under the const folder, I have a default URL set to return back the same results if there are no filters selected. The only possible way to grab the results is by selecting at least one of the provided parameters specified by API Ninja. So I carefully selected to give a list of dogs with a minimum height of 6 inches as an adult.

Reply:
it took me a while to see what you meant and I know the error that the plus(+) symbol was unnecessary. I corrected it the URL structure